### PR TITLE
Add an extra hint for assimp.

### PR DESCRIPTION
**Pull request by drwells**
If assimp has its own directory, e.g., `ASSIMP_DIR=/opt/assimp/`, then this hint is needed.

### CODE REVIEW

File: cmake/modules/FindASSIMP.cmake [hunk 1]
```diff
@@ -31,7 +31,7 @@ DEAL_II_FIND_LIBRARY(ASSIMP_LIB NAMES assimp
   )
 
 DEAL_II_FIND_PATH(ASSIMP_INC assimp/defs.h
-  HINTS ${ASSIMP_DIR}
+  HINTS ${ASSIMP_DIR} ${ASSIMP_DIR}/include/
```
Comment (tamiko) [inline cmake/modules/FindASSIMP.cmake]: One nit-pick: Can you refactor to:
```
DEAL_II_FIND_PATH(ASSIMP_INC assimp/defs.h
  HINTS ${ASSIMP_DIR}
  PATH_SUFFIXES include
```
?
And quickly check :-)

Comment (drwells) [reply cmake/modules/FindASSIMP.cmake]: sure

Comment (drwells) [reply cmake/modules/FindASSIMP.cmake]: done

**Comment by masterleinad** [comment]
Yep, that works.

**Pull request merged**